test/images/webhook/main.go

```diff
@@ -44,8 +44,9 @@ const (
 
 // Config contains the server (the webhook) cert and key.
 type Config struct {
-	CertFile string
-	KeyFile  string
+	CertFile     string
+	KeyFile      string
+	ClientCAFile string
```

Comment:
I would add some separation in this struct, and maybe a comment; otherwise a random person will think these are the normal key/cert/ca triplet when they actually aren't. Something like: "Cert/Key are the server cert which this server presents when receiving requests. ClientCA is used to verify that any client cert presented by the caller of this webhook is trusted (it must be signed by this CA)."

Reply:
separation like an empty line? or multiple structs

Reply:
Empty line is what I had in mind. But anything that gets the point across is fine :)

```go
type Config struct {
	// Comment about these two fields
	CertFile string
	KeyFile  string

	// Comment about this field
	ClientCAFile string
}
```